homeassistant/components/sensor/xiaomi_aqara.py

```diff
@@ -26,7 +26,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
                                             'temperature', gateway))
                 devices.append(XiaomiSensor(device, 'Humidity',
                                             'humidity', gateway))
-            elif device['model'] == 'weather.v1':
+            elif device['model'] in ['weather', 'weather.v1']:
```

Comment:
Are you sure about this ("weather") model name?

Reply:
yes ,i sure